Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -5,6 +5,7 @@ import Prelude
 import ReactiveExtensions
 import ReactiveExtensions_TestHelpers
 @testable import Stripe
```

Comment:
Good point - tested removing `import Stripe` in favor of `import StripePaymentSheet` and tests still pass.
Corrections made.

Reply:
@scottkicks I wanted to call out this mistake in your last [pr](https://github.com/kickstarter/ios-oss/pull/1833) that caused a build crash. I only noticed it when trying to build the this branch to run on device.

It's easy to miss in the review because its' a change to the `.project.pbxproj` file, which is a massive directory basically so hard to track file membership of which file does in which framework. 

Turns out `GraphAPI.TriggerThirdPartyEventInput+TriggerThirdPartyEventInputTests.swift` and `TriggerThirdPartyEventInputTests.swift` were added to `KsApi` when they should be added to `KsApiTests`

Fixed [here](https://github.com/kickstarter/ios-oss/pull/1834/commits/6bf56866051654bb4ba3b451d2f86000b4de2902).

I had to go back to my fix graph schema pr, run it on device, see that it was working then work backwards to find that the pr above had an issue with file membership which caused the on-launch crash.

Just be careful with adding the files to the right frameworks because it effects the run-time scenario in the app, even if the code compiles. Not sure why tests still passed in CI though.